src/main/java/com/capitalone/dashboard/model/adapter/BuildStageAdapter.java

```diff
@@ -26,7 +27,7 @@ public BuildStage deserialize(JsonElement jsonElement, Type type, JsonDeserializ
         buildStage.setEndTime(getStringValue(jsonObject, "endTime"));
         buildStage.setLog(getStringValue(jsonObject, "log"));
         buildStage.setParentId(getStringValue(jsonObject, "parentId"));
-        buildStage.setError(getStringValue(jsonObject, "error"));
+        buildStage.setError((Error) getObjectValue(jsonObject, "error"));
```

Comment:
can you change this to buildStage.setError((Error) jsonObject.get("error")));

Reply:
or better way is to use GSON to build the Error Object

Reply:
https://static.javadoc.io/com.google.code.gson/gson/2.6.2/com/google/gson/Gson.html#fromJson-com.google.gson.JsonElement-java.lang.reflect.Type-